src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -333,23 +351,18 @@ def _missing_deps_finder(self):
     return MissingDependencyFinder(dep_analyzer, CompileErrorExtractor(
       self.get_options().class_not_found_error_patterns))
 
-  def _compile_context(self, target, target_workdir):
-    analysis_file = os.path.join(target_workdir, 'z.analysis')
-    classes_dir = os.path.join(target_workdir, 'classes')
-    jar_file = os.path.join(target_workdir, 'z.jar')
-    log_dir = os.path.join(target_workdir, 'logs')
-    zinc_args_file = os.path.join(target_workdir, 'zinc_args')
+  def create_compile_context(self, target, target_workdir):
     return CompileContext(target,
-                          analysis_file,
-                          classes_dir,
-                          jar_file,
-                          log_dir,
-                          zinc_args_file,
-                          self._compute_sources_for_target(target))
+      os.path.join(target_workdir, 'z.analysis'),
```

Comment:
I agree. I need to figure out how to get Intellij's autoformat style in line with expectations so that I can use its refactoring tooling without leaving weird whitespace like this everywhere.